manifest.yml

```diff
@@ -2,13 +2,9 @@
 applications:
 - name: pulse
   memory: 512M
-  instances: 1
+  instances: 2
   buildpack: python_buildpack
 - name: pulse-staging
-  memory: 512M
-  instances: 1
-  buildpack: python_buildpack
-- name: pulse-subdomains
-  memory: 512M
+  memory: 128M
```

Comment:
Diff here is confusing, but it's deleting the `pulse-subdomains` app (not needed, it's in production now), and dropping the `pulse-staging` memory down to 128M (it uses way less than that).